apps/bfd-server/dev/bfd-data-dictionary/data/V2/0125_hcpcs_1st_mdfr_cd.json

```diff
@@ -39,7 +39,7 @@
             "additional": [],
             "derived": "",
             "note": "The eob.item[N].modifier[N].coding[N].version is set by the hcpcs year.",
-            "example": "[\\n  {\\n    \"coding\": [\\n      {\\n        \"system\": \"https://bluebutton.cms.gov/resources/codesystem/hcpcs\",\\n        \"version\": \"5\",\\n        \"code\": \"LT\"\\n      }\\n    ]\\n  }\\n]"
+            "example": "[\\n  {\\n    \"system\": \"https://bluebutton.cms.gov/resources/codesystem/hcpcs\",\\n    \"version\": \"5\",\\n    \"code\": \"LT\"\\n  }\\n]\\n"
```

Comment:
Am looking at this, but need to check changes vs. FHIR spec, etc. Maybe you can short-circuit my angst...wondering about removal of parent "coding" in this...what happens if there are multiple codings?

Reply:
It's strange too because I notice that 0126-0128 have the  same pattern but weren't updated in the Confluence doc.  Should I revert this one?

Reply:
I've reverted this change